scapy/pton_ntop.py

```diff
@@ -18,9 +18,29 @@
 from scapy.compat import *
 
 _IP6_ZEROS = re.compile('(?::|^)(0(?::0)+)(?::|$)')
+_INET4_PTON_EXC = socket.error("illegal IP address string passed to _inet4_pton")  # noqa: E501
 _INET6_PTON_EXC = socket.error("illegal IP address string passed to inet_pton")
 
 
+def _inet4_pton(addr):
```

Comment:
As seen in the comments, this function should be removed.

Reply:
Then there's nothing left in the PR ? (except the test)

Reply:
Shall we close it?